### PR TITLE
More copy editing for glossary.md

### PR DESCRIPTION
**Pull request by chrishtr**

### CODE REVIEW

File: book/glossary.md [hunk 1]
```diff
@@ -472,17 +471,17 @@ the computer or web browser. The most common are screen readers.
...
 
-: Central Processing Unit, the hardware component in a computer that
+: Central processing unit, the hardware component in a computer that
 executes generic compute programs.
 
 DukPy
 
 : A Python binding library for Duktape, a JavaScript engine used in
-this book.
+this book.]
```
Comment (pavpanchekha) [inline book/glossary.md:480]: Seems wrong

**Review by pavpanchekha — APPROVED** [review]
Please fix & then merge

**Comment by pavpanchekha** [comment]
The definition for "Decode" also has "an an", please fix before merging.

File: book/glossary.md (hunk 1 above)
Comment (chrishtr) [inline book/glossary.md:480]:
```suggestion
this book.
```

File: book/glossary.md [hunk 2]
```diff
@@ -379,7 +378,7 @@ surfaces on a GPU, in order to speed up animations.
 
 Decode
 
-: converting from a compressed format for a resource (such as an 
+: Converting from a compressed format for a resource (such as an
```
Comment (chrishtr) [inline book/glossary.md:381]:
```suggestion
: Converting from a compressed format for a resource (such as
```

**Pull request merged**